### PR TITLE
Fix new clippy lints

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes
- [ ] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

File: bee-api/bee-rest-api/src/endpoints/config.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ pub(crate) const DEFAULT_FEATURE_PROOF_OF_WORK: bool = true;
 pub(crate) const DEFAULT_WHITE_FLAG_SOLIDIFICATION_TIMEOUT: Duration = Duration::from_secs(2);
 
 /// REST API configuration builder.
-#[derive(Default, Deserialize, PartialEq)]
+#[derive(Default, Deserialize, PartialEq, Eq)]
```
Comment (Thoralf-M) [inline bee-api/bee-rest-api/src/endpoints/config.rs]: Order

Comment (thibault-martinez) [reply bee-api/bee-rest-api/src/endpoints/config.rs]: Hmm yeah I can but overall they are not ordered :/

Comment (Thoralf-M) [reply bee-api/bee-rest-api/src/endpoints/config.rs]: Ok do what you want

**Review by Thoralf-M — DISMISSED** [review]

File: bee-api/bee-rest-api/src/endpoints/config.rs (hunk 1 above)
Comment (thibault-martinez) [reply bee-api/bee-rest-api/src/endpoints/config.rs]: I've done it all good 👍🏻

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**